pycbc/inference/models/relbin.py

```diff
@@ -338,22 +346,19 @@ def _lognl(self):
         return 0.
```

Comment:
I think both these methods (`_loglikelihood` and `_lognl`) can be removed now. I had avoided implementing the full `_lognl` calculation when I wrote this class because it would've required replicating a lot of the data handling code in `GaussianNoise`, but now that you've moved it all to the new class which is inherited by `RelativeSPA` we should just be able to inherit the `_lognl` method from there. Similarly the `_loglikelihood` method here is just replicating what you now have in `BaseGaussianNoise`

Reply:
Thanks for catching @dfinstad, I missed those two functions. I've removed them and updated.